<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>946</width>
    <height>498</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>946</width>
    <height>498</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>946</width>
    <height>498</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Данные метрологии</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>931</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Просмотр мероприятий</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>911</width>
        <height>261</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Название мероприятия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата и время</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Место проведения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Выполнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ответственный</string>
       </property>
      </column>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>911</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Введите фамилию и инициалы:</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>771</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>20</y>
         <width>111</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ввести</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбранная запись:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>330</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Выполнена</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Добавления записи</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>10</y>
        <width>331</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавление записи</string>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Название мероприятия:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>190</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Дата и время:</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2023</year>
         <month>1</month>
         <day>2</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2023</year>
         <month>1</month>
         <day>2</day>
        </date>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Место проведения:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ответственный:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>150</y>
         <width>211</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>341</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Название мероприятия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата и время</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Место проведения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Выполнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ответственный</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Информация о выполнение/невыполения</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>451</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Название мероприятия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата и время</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Место проведения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Выполнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ответственный</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableWidget_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>451</width>
        <height>311</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>Название мероприятия</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата и время</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Место проведения</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Выполнение</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ответственный</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Выполненные мероприятия</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>441</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Невыполненные мероприятия/Просрочнно выполнение</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>461</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Подключение к серверу:</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP-адрес:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Порт:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>5555</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Подключить</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>420</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Отсортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>460</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести отчет 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Вывести отчет 2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
